syntax = "proto3";

package nvidia.riva;

enum AudioEncoding {
  ENCODING_UNSPECIFIED = 0;
  LINEAR_PCM = 1;
  FLAC = 2;
  MULAW = 3;
  ALAW = 4;
  OGGOPUS = 5;
}